src/Illuminate/Database/Console/DatabaseInspectionCommand.php

```diff
@@ -22,15 +17,7 @@ abstract class DatabaseInspectionCommand extends Command
      */
     protected function getConnectionName(ConnectionInterface $connection, $database)
     {
-        return match (true) {
-            $connection instanceof MySqlConnection && $connection->isMaria() => 'MariaDB',
-            $connection instanceof MySqlConnection => 'MySQL',
-            $connection instanceof MariaDbConnection => 'MariaDB',
```

Comment:
Bugfix: `MariaDbConnection` extends `MySqlConnection`, so "MariaDB" is never shown currently.